src/helpers/response_message.coffee

```diff
@@ -0,0 +1,69 @@
+class ResponseMessage
+
+  teamCreated: (team)->
+    "#{team.label()} created, add some people to it"
+
+  teamAlreadyExists: (teamName)->
+    "#{teamName} team already exists"
+
+  teamDeleted: (teamName)->
+    "Team `#{teamName}` removed"
+
+  listTeams: (teams)->
+    return 'No team was created so far' if teams.length is 0
+    message = "Teams:\n"
+
+    for team in teams
+      if team.membersCount() > 0
+        message += "`#{team.name}` (#{team.membersCount()} total)"
+
+        for user in team.members
+          message += "\n- #{user}"
+      else
+        message += "`#{team.name}` (empty)\n"
+    message
+
+  adminRequired: -> "Sorry, only admins can remove teams :)"
```

Comment:
Since you use this message for both clearing and deleting teams, just say `Sorry, only admins can perform this operation` instead of having two distinct messages as there were before.